doajtest/unit/test_tasks_ingestCrossref531Articles.py

```diff
@@ -814,6 +824,7 @@ def test_27_run_exists(self):
         fu = models.FileUpload.pull(id)
         assert fu is not None
         assert fu.status == "processed"
+        assert job.outcome_status == BgjobOutcomeStatus.Pending
```

Comment:
should we use 'value' attribute for Pending?

Reply:
yes, missing `.value`